talmSubscription.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
This CR has the right content, but should be in the path telco-core/telco-hub/configuration/reference-crs/required/talm/talmSubscription.yaml. Right now this PR has 2 copies of this CR and should only have the one.